### DISCUSSION TITLE
MusicXML files not being loaded properly

### DISCUSSION THREAD

**Discussion by mxAlexG** [CoderLine/alphaTab#861] [Q&A] [answered]
Hi everyone,

I tried to build the project (develop branch) and it all went well for when loading guitarpro files. The problem however occurs when I attempt to load musicxml files (the xml file within). This problem never arises when using the official distributables (both alphaTab.js and alphaTab.min.js).

The log from the console:
![image](https://user-images.githubusercontent.com/105655048/168615078-4b38f3d6-d244-40a4-9085-ec3747126558.png)

Unchecked runtime.lastError: The message port closed before a response was received.
player.html:1 Unchecked runtime.lastError: The message port closed before a response was received.
player.html:1 Unchecked runtime.lastError: The message port closed before a response was received.
alphaTab.js:25786 The AudioContext was not allowed to start. It must be resumed (or created) after a user gesture on the page. https://goo.gl/7K7WLu
createAudioContext @ alphaTab.js:25786
open @ alphaTab.js:25795
open @ alphaTab.js:26576
AlphaSynthWebWorkerApi @ alphaTab.js:25998
createWorkerPlayer @ alphaTab.js:27065
setupPlayer @ alphaTab.js:24594
AlphaTabApiBase @ alphaTab.js:24246
AlphaTabApi @ alphaTab.js:27272
(anonymous) @ player.html:350
alphaTab.js:2868 Uncaught TypeError: Cannot set properties of undefined (setting 'slurDestination')
    at Note.chain (alphaTab.js:2868)
    at Beat.chain (alphaTab.js:3992)
    at Voice$1.chain (alphaTab.js:5209)
    at Voice$1.finish (alphaTab.js:5237)
    at Bar.finish (alphaTab.js:400)
    at Staff.finish (alphaTab.js:5022)
    at Track.finish (alphaTab.js:5112)
    at Score.finish (alphaTab.js:4612)
    at Function.jsObjectToScore (alphaTab.js:20596)
    at AlphaTabWebWorker.handleMessage (alphaTab.js:21865)
chain @ alphaTab.js:2868
chain @ alphaTab.js:3992
chain @ alphaTab.js:5209
finish @ alphaTab.js:5237
finish @ alphaTab.js:400
finish @ alphaTab.js:5022
finish @ alphaTab.js:5112
finish @ alphaTab.js:4612
jsObjectToScore @ alphaTab.js:20596
handleMessage @ alphaTab.js:21865

**Comment by Danielku15** [comment, marked as answer]
Could be a new issue introduced in 1.3 which is not yet considered fully stable. Can you provide the MusicXML file you used in your tests? Despite the "Standard Status" of MusicXML, each software still expresses certain things different in the files. Therefore I need to have a dedicated look at the files used in your tests.

**Reply by mxAlexG** [reply]
I used some random scores I downloaded from musescore.org. The issue persists regardless of which score file I use. On the other hand,  the dist files built  from master don't show such issue when used.
